### ISSUE TITLE
Add a high contrast mode

### ISSUE THREAD

**Issue by b-x-wu** [b-x-wu/b-x-wu.github.io#1]
Right now, the color scheme does not meet [WCAG's qualifications for color contrast accessibility](https://www.w3.org/WAI/WCAG21/Techniques/general/G174.html). In order to meet this bar, we either need to swap out the color scheme, or add in a way to activate a high contrast mode.

Resources
- https://ingosteinke.medium.com/accessible-to-a-making-a-pastel-color-scheme-accessible-by-adding-a-high-contrast-alternative-534e7a8a56af
- https://stackoverflow.com/questions/72547565/how-to-provide-an-accessible-high-contrast-alternative-to-a-pastel-color-scheme

**Comment by b-x-wu** [comment]
Resolved with #6
